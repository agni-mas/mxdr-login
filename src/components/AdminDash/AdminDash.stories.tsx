/* eslint-disable */
import AdminDash from './AdminDash';

export default {
  title: "AdminDash",
};

export const Default = () => <AdminDash />;

Default.story = {
  name: 'default',
};
