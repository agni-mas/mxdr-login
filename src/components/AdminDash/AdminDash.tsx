import React, { FC } from 'react';
import GridPageLayout from '../global/GridLayout/GridPageLayout';
import Grid from '../layout/Grid/Grid';
import UsersAssigned from './UsersAssigned/UsersAssigned';
import Button from '@mui/material/Button';
import AddIcon from '@mui/icons-material/Add';
import SettingsIcon from '@mui/icons-material/Settings';
import HealthStatus from './HealthStatus/HealthStatus';
import OnboardingStatus from './OnboardingStatus/OnboardingStatus';
import EscalationContacts from './EscalationContacts/EscalationContacts';
import NewAlertsAndReports from './NewAlertsAndReports/NewAlertsAndReports';
import ServiceSummary from '../AnalystDash/ServiceSummary/ServiceSummary';
import Entitlements from './Entitlements/Entitlements';
import GridItem from '../layout/GridItem/GridItem';

interface userData {
  firstName: string
  lastName: string
  email: string
  role: string
  organization: string
  lastLogin: string
}

interface AdminDashProps {}

const styles = {
  topButtons: {
    display: 'flex',
    justifyContent: 'flex-end',
    marginTop: 35,
  },
  header: {
    fontFamily: 'Graphik',
    fontStyle: 'normal',
    fontWeight: 300,
    fontSize: 22,
    color: 'white',
  },
  button: {
    backgroundColor:'#7500C0',
    border: 'none',
    marginRight: 10,
  },
  icon: {
    paddingLeft: 5,
  },

}

const gridBoxData = [
  { num: 27, type: "Assigned to Me", isSelected: false },
  { num: 58, type: "Assigned to Org", isSelected: true },
  { num: 6, type: "Asset Alarms", isSelected: false },
  { num: 3, type: "Critical Alerts Today", isSelected: false },
  { num: 11, type: "Critical Incidents Today", isSelected: false },
];

const AdminDash: FC<AdminDashProps> = () => {
  return (
    <section>
      <GridPageLayout {...gridBoxData} />
      <div style={styles.topButtons}>
      <Button color='secondary' variant="outlined" style={styles.button}>New Role<AddIcon style={styles.icon} /></Button>
      <Button color='secondary' variant="outlined">Customize<SettingsIcon style={styles.icon} /></Button>
      </div>
    <Grid>
      <UsersAssigned />
      <HealthStatus />
      <Entitlements />
      <OnboardingStatus />
      <EscalationContacts />
      <NewAlertsAndReports />
      <GridItem title='Service Summary (Last 7 days) '>
        <ServiceSummary/>
      </GridItem>
      {/* <AlertDetectionAndResponse /> */}
      {/* {children} */}
    </Grid>
    </section>
  )
};


export default AdminDash;
