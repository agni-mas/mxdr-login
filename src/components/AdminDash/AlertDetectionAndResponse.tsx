import react, { FC } from 'react';
import GridItem from '../layout/GridItem/GridItem';
import Button from '@mui/material/Button';
import AddIcon from '@mui/icons-material/Add';
import Table from '../global/table/Table';
import FormGroup from '@mui/material/FormGroup';
import FormControlLabel from '@mui/material/FormControlLabel';
import Checkbox from '@mui/material/Checkbox';


const styles = {
    
}

interface AlertDetectionAndResponseProps {}

const AlertDetectionAndResponse: FC<AlertDetectionAndResponseProps> = () => (
    <GridItem title="Alert Detection and Response">
    </GridItem>
);

export default AlertDetectionAndResponse;