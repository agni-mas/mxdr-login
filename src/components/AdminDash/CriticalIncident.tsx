import react, { FC } from 'react';
import GridItem from '../layout/GridItem/GridItem';
import Button from '@mui/material/Button';
import AddIcon from '@mui/icons-material/Add';
import Table from '../global/table/Table';
import FormGroup from '@mui/material/FormGroup';
import FormControlLabel from '@mui/material/FormControlLabel';
import Checkbox from '@mui/material/Checkbox';
import CriticalIncidentTable from '../global/table/Tables/CriticalIncidentTable';


const styles = {
   
}

interface IncidentDetectionProps {}

const IncidentDetection: FC<IncidentDetectionProps> = () => (
    <GridItem title="Critical Incident Trends (last 7 days)">
        <CriticalIncidentTable />
    </GridItem>
);

export default IncidentDetection;