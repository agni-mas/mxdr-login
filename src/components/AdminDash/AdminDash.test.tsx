import React from 'react';
import { render } from "@testing-library/react";
import AdminDash from './AdminDash';


it("renders <AdminDash />", () => {
  const { asFragment } = render(<AdminDash />);
  expect(asFragment()).toMatchSnapshot();
});
