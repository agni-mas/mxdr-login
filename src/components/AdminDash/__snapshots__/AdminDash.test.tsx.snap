// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <AdminDash /> 1`] = `
<DocumentFragment>
  <section>
    <div
      class="MuiBox-root css-i9gxme"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true css-1vd824g-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-8nny99-MuiPaper-root"
          >
            <div
              style="padding-top: 13px; padding-left: 24px; padding-bottom: 13px;"
            >
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 32px; padding-bottom: 6px;"
              >
                27
              </div>
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 14px;"
              >
                Assigned to Me
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true css-1vd824g-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-8nny99-MuiPaper-root"
          >
            <div
              style="padding-top: 13px; padding-left: 24px; padding-bottom: 13px; background-color: rgb(161, 0, 255);"
            >
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 32px; padding-bottom: 6px;"
              >
                58
              </div>
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 14px;"
              >
                Assigned to Org
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true css-1vd824g-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-8nny99-MuiPaper-root"
          >
            <div
              style="padding-top: 13px; padding-left: 24px; padding-bottom: 13px;"
            >
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 32px; padding-bottom: 6px;"
              >
                6
              </div>
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 14px;"
              >
                Asset Alarms
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true css-1vd824g-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-8nny99-MuiPaper-root"
          >
            <div
              style="padding-top: 13px; padding-left: 24px; padding-bottom: 13px;"
            >
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 32px; padding-bottom: 6px;"
              >
                3
              </div>
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 14px;"
              >
                Critical Alerts Today
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true css-1vd824g-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-8nny99-MuiPaper-root"
          >
            <div
              style="padding-top: 13px; padding-left: 24px; padding-bottom: 13px;"
            >
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 32px; padding-bottom: 6px;"
              >
                11
              </div>
              <div
                style="font-family: Graphik; font-style: normal; font-weight: 400; font-size: 14px;"
              >
                Critical Incidents Today
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      style="display: flex; justify-content: flex-end; margin-top: 35px;"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-wnpvqa-MuiButtonBase-root-MuiButton-root"
        style="background-color: rgb(117, 0, 192); margin-right: 10px;"
        tabindex="0"
        type="button"
      >
        New Role
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="AddIcon"
          focusable="false"
          style="padding-left: 5px;"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
          />
        </svg>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-wnpvqa-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        Customize
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="SettingsIcon"
          focusable="false"
          style="padding-left: 5px;"
          viewBox="0 0 24 24"
        >
          <path
            d="M19.14 12.94c.04-.3.06-.61.06-.94 0-.32-.02-.64-.07-.94l2.03-1.58c.18-.14.23-.41.12-.61l-1.92-3.32c-.12-.22-.37-.29-.59-.22l-2.39.96c-.5-.38-1.03-.7-1.62-.94l-.36-2.54c-.04-.24-.24-.41-.48-.41h-3.84c-.24 0-.43.17-.47.41l-.36 2.54c-.59.24-1.13.57-1.62.94l-2.39-.96c-.22-.08-.47 0-.59.22L2.74 8.87c-.12.21-.08.47.12.61l2.03 1.58c-.05.3-.09.63-.09.94s.02.64.07.94l-2.03 1.58c-.18.14-.23.41-.12.61l1.92 3.32c.12.22.37.29.59.22l2.39-.96c.5.38 1.03.7 1.62.94l.36 2.54c.05.24.24.41.48.41h3.84c.24 0 .44-.17.47-.41l.36-2.54c.59-.24 1.13-.56 1.62-.94l2.39.96c.22.08.47 0 .59-.22l1.92-3.32c.12-.22.07-.47-.12-.61l-2.01-1.58zM12 15.6c-1.98 0-3.6-1.62-3.6-3.6s1.62-3.6 3.6-3.6 3.6 1.62 3.6 3.6-1.62 3.6-3.6 3.6z"
          />
        </svg>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      class="grid"
    >
      <section
        class="item"
      >
        <h2
          class="grid-item-title"
        >
          Users Assigned To Organization
        </h2>
        <div
          style="display: flex; justify-content: space-between; align-items: center; padding-bottom: 20px;"
        />
        <div
          class="top-controls"
        >
          <div
            class="css-w4z10b-MuiStack-root"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium hover-state css-v87ghe-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Users (0) 
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium hover-state css-v87ghe-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Roles (0) 
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium hover-state css-v87ghe-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Org Profile (0) 
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textSecondary MuiButton-sizeMedium MuiButton-textSizeMedium css-1d3f8j8-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Filter By
            <span
              class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="FilterAltIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M4.25 5.61C6.27 8.2 10 13 10 13v6c0 .55.45 1 1 1h2c.55 0 1-.45 1-1v-6s3.72-4.8 5.74-7.39c.51-.66.04-1.61-.79-1.61H5.04c-.83 0-1.3.95-.79 1.61z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
        >
          <table
            aria-label="incidentId table"
            class="MuiTable-root border table css-rqglhn-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-ubb257-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1p6fnva-MuiTableCell-root"
                  scope="col"
                >
                  First Name
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Last Name
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Business Email
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-hsoe6c-MuiTableCell-root"
                  scope="col"
                >
                  Roles
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-hsoe6c-MuiTableCell-root"
                  scope="col"
                >
                  Organization
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-hsoe6c-MuiTableCell-root"
                  scope="col"
                >
                  Last Log In
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            />
          </table>
        </div>
        <div
          class="bottom-control"
        >
          <div
            class="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
              style="color: white; font-family: Graphik; font-style: normal; font-weight: 500;"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorDefault PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorDefault MuiCheckbox-root MuiCheckbox-colorDefault css-dk24d0-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxOutlineBlankIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Show Deactivated
              </span>
            </label>
          </div>
          <span
            class="row-count"
          >
            Showing: 0 of 0
          </span>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-16d6pf0-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            View All
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </section>
      <section
        class="item"
      >
        <h2
          class="grid-item-title"
        >
          Health Status
        </h2>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-disableElevation MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-disableElevation css-jqgntc-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Device Health
          <span
            class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="KeyboardArrowDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7.41 8.59 12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <span
          class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-wave css-1kcnzpi-MuiSkeleton-root"
          style="height: 200px;"
        />
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-wnpvqa-MuiButtonBase-root-MuiButton-root"
          style="float: right; border-radius: 28px;"
          tabindex="0"
          type="button"
        >
          View Details
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </section>
      <section
        class="item"
      >
        <h2
          class="grid-item-title"
        >
          Entitlements (last 7 days)
        </h2>
        <div
          class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
        >
          <table
            aria-label="entitlements table"
            class="MuiTable-root border table css-rqglhn-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-ubb257-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-izw8y7-MuiTableCell-root"
                  scope="col"
                >
                  Entitlement Search Code
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Description
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Start Date
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  End Date
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Purchased
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Used
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-hsoe6c-MuiTableCell-root"
                  scope="col"
                >
                  Available
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            />
          </table>
        </div>
        <div
          class="MuiTableContainer-root css-pdynqw-MuiTableContainer-root"
        >
          <table
            aria-label="entitlements table"
            class="MuiTable-root border table css-rqglhn-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-2poe3x-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-izw8y7-MuiTableCell-root"
                  scope="col"
                >
                  Device Type
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Used
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            />
          </table>
        </div>
      </section>
      <section
        class="item"
      >
        <h2
          class="grid-item-title"
        >
          Onboarding Status
        </h2>
        <span
          class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-wave css-1kcnzpi-MuiSkeleton-root"
          style="height: 200px;"
        />
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-wnpvqa-MuiButtonBase-root-MuiButton-root"
          style="float: right; border-radius: 28px;"
          tabindex="0"
          type="button"
        >
          View Details
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </section>
      <section
        class="item"
      >
        <h2
          class="grid-item-title"
        >
          Escalation Contacts
        </h2>
        <div
          class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
        >
          <table
            aria-label="incidentId table"
            class="MuiTable-root border table css-rqglhn-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-ubb257-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                  scope="col"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                    data-testid="GroupIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5c-1.66 0-3 1.34-3 3s1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5C6.34 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z"
                    />
                  </svg>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Contacts
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
                  scope="col"
                >
                  Role
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-hsoe6c-MuiTableCell-root"
                  scope="col"
                >
                  Location
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-rmn4b8-MuiTableCell-root"
                  scope="col"
                >
                  Connect
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            />
          </table>
        </div>
      </section>
      <section
        class="item"
      >
        <h2
          class="grid-item-title"
        >
          New Alerts and Reports
        </h2>
        <div
          class="alerts"
        >
          <div
            class="MuiBox-root css-1gsv261"
          >
            <div
              class="MuiTabs-root css-1ujnqem-MuiTabs-root"
            >
              <div
                class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
                style="overflow: hidden; margin-bottom: 0px;"
              >
                <div
                  aria-label="basic tabs example"
                  class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
                  role="tablist"
                >
                  <button
                    aria-controls="simple-tabpanel-0"
                    aria-selected="true"
                    class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                    color="secondary"
                    id="simple-tab-0"
                    role="tab"
                    tabindex="0"
                    type="button"
                  >
                    Alerts(0)
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                  <button
                    aria-controls="simple-tabpanel-1"
                    aria-selected="false"
                    class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                    id="simple-tab-1"
                    role="tab"
                    tabindex="-1"
                    type="button"
                  >
                    Reports (0)
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
                <span
                  class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
                  style="left: 0px; width: 0px;"
                />
              </div>
            </div>
          </div>
          <span
            class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-wave css-1kcnzpi-MuiSkeleton-root"
            style="height: 400px;"
          />
          <div
            aria-labelledby="simple-tab-1"
            hidden=""
            id="simple-tabpanel-1"
            role="tabpanel"
          />
        </div>
      </section>
      <section
        class="item"
      >
        <h2
          class="grid-item-title"
        >
          Service Summary (Last 7 days) 
        </h2>
        <div
          class="bars"
        >
          <div>
            <h3
              class="bar-chart-title"
            >
              Incidents by urgency
            </h3>
            <div
              class="recharts-responsive-container"
              style="width: 100%; height: 250px;"
            />
          </div>
          <div>
            <h3
              class="bar-chart-title"
            >
              Requests by urgency
            </h3>
            <div
              class="recharts-responsive-container"
              style="width: 100%; height: 250px;"
            />
          </div>
        </div>
      </section>
    </div>
  </section>
</DocumentFragment>
`;
